tardis/apps/publication_workflow/components/PublicationButton.jsx

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+import React, { Fragment } from 'react';
+
+const PublicationButton = ({ onclick, setViewType }) => (
+  <Fragment>
+    <div className="row">
+      <div className="col-md-12 pull-right">
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 12 spaces but found 6.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
            <div className="col-md-12 pull-right">
```